consensus/state_sync/helpers.go

```diff
@@ -6,6 +6,25 @@ import (
 	"google.golang.org/protobuf/types/known/anypb"
 )
 
+// TODO(#352): Implement this function, currently a placeholder.
+// Helper function for broadcasting state sync messages to the all peers known to the node
+// It is used for:
+//
+//		requesting for metadata, via the periodicMetaDataSynch() function
```

Comment:
1. Metadata is one word -> `MetaData` -> `Metadata`
2. An easier read: `requests for metadata using `periodicMetadataSync`; ditto below